### DISCUSSION TITLE
v1.36.1

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1346] [General]
## Minor update
- Fixed issue with sessions being loaded incorrectly
- Language translation updates

A huge thanks to all the contributors on this release! @Lemonawa @RikoDEV @KevinVR

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v1.36.1'>v1.36.1</a>.</em>

**Comment by KonstantinReido** [comment]
@mikecao faced with an issue https://github.com/umami-software/umami/issues/1071#issuecomment-1197318133, decided to duplicate it here because worry that the original ticket is already closed and you can miss it.
